src/GitHub.App/ViewModels/PullRequestListViewModel.cs

```diff
@@ -61,39 +103,38 @@ public class PullRequestListViewModel : BaseViewModel, IPullRequestListViewModel
                 VisualStudio.Services.DefaultExportProvider.GetExportedValue<IVisualStudioBrowser>().OpenUrl(repositoryHost.Address.WebUri);
             });
 
-            States = new List<PullRequestState> {
-                new PullRequestState { IsOpen = true, Name = "Open" },
-                new PullRequestState { IsOpen = false, Name = "Closed" },
-                new PullRequestState { Name = "All" }
-            };
-
             trackingAuthors = new TrackingCollection<IAccount>(Observable.Empty<IAccount>(),
                 OrderedComparer<IAccount>.OrderByDescending(x => x.Login).Compare);
             trackingAssignees = new TrackingCollection<IAccount>(Observable.Empty<IAccount>(),
                 OrderedComparer<IAccount>.OrderByDescending(x => x.Login).Compare);
+
             trackingAuthors.Subscribe();
             trackingAssignees.Subscribe();
 
             Authors = trackingAuthors.CreateListenerCollection(EmptyUser, this.WhenAnyValue(x => x.SelectedAuthor));
             Assignees = trackingAssignees.CreateListenerCollection(EmptyUser, this.WhenAnyValue(x => x.SelectedAssignee));
 
             PullRequests = new TrackingCollection<IPullRequestModel>();
-            pullRequests.Comparer = OrderedComparer<IPullRequestModel>.OrderByDescending(x => x.UpdatedAt).Compare;
             pullRequests.NewerComparer = OrderedComparer<IPullRequestModel>.OrderByDescending(x => x.UpdatedAt).Compare;
 
             this.WhenAny(x => x.SelectedState, x => x.Value)
-                .Where(x => PullRequests != null)
+                .Where(x => pullRequests != null)
```

Comment:
Out of interest @shana why did you change these to use the underlying field rather than the property?